ChangeLog.md

```diff
@@ -17,6 +17,9 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- Added new linker option -s HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS=0
```

Comment:
`=0` here, but the next line says "default enabled"?

Reply:
Yes, deferring is still enabled by default so that all those SDL/SDL2 etc code will continue to work as usual. People should explicitly opt in for disabling deferring support.

Reply:
I agree, but the changelog text here looks incorrect. Shouldn't the `=0` be removed? Or am I misunderstanding the intention here?

Reply:
Default value is for this setting to be enabled (`var HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS=1` in src/settings.js), i.e. deferring requests is supported. If user passes `-s HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS=0` he can disable the support. If user passed `-s HTML5_SUPPORT_DEFERRING_USER_SENSITIVE_REQUESTS`, then he'd be explicitly stating the default (enable deferring support).

Reply:
Ok, I think the changelog text is a little ambiguous, but maybe it's just me...